README.md

```diff
@@ -1,7 +1,13 @@
+# Added today
```

Comment:
👍 for the link to Gitpod, but let's remove `# Added today` title.

Reply:
Done